File: PlayingAudio/app/src/main/java/com/vkt4u9999/playingaudio/MainActivity.java
package com.vkt4u9999.playingaudio;

import androidx.appcompat.app.AppCompatActivity;

import android.media.AudioManager;
import android.media.MediaPlayer;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.ImageButton;
import android.widget.SeekBar;

public class MainActivity extends AppCompatActivity {

    MediaPlayer mediaPlayer;
    Button button;
    SeekBar volumeSeekBar;
    AudioManager audioManager;       // Дает доступ к звуку

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        // Устанавливаем звук на все активити
        mediaPlayer = MediaPlayer.create(getApplicationContext(), R.raw.taras);

        button = findViewById(R.id.button);
        volumeSeekBar = findViewById(R.id.volumeSeekBar);
        audioManager = (AudioManager) getSystemService(AUDIO_SERVICE);


        // Получаем данные о макс громкости
        int maxVolume = audioManager.getStreamMaxVolume(AudioManager.STREAM_MUSIC);
        volumeSeekBar.setMax(maxVolume);

        volumeSeekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
            @Override
            public void onProgressChanged(SeekBar seekBar, int i, boolean b) {
                Log.d("Progress changed: ", "" + i);
                audioManager.setStreamVolume(AudioManager.STREAM_MUSIC, i, 0);

            }

            @Override
            public void onStartTrackingTouch(SeekBar seekBar) {

            }

            @Override
            public void onStopTrackingTouch(SeekBar seekBar) {

            }
        });

       button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if (mediaPlayer.isPlaying()){
                    pause();
                }
                else{
                    play();
                }
            }
        });

    }
    public void play(){
        mediaPlayer.start();
        button.setText("Pause");
    }

    public void pause(){
        mediaPlayer.pause();
        button.setText("Play");
    }
    }

File: MyApp/app/src/main/java/com/vkt4u9999/myapp/MainActivity.kt
package com.vkt4u9999.myapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import androidx.fragment.app.Fragment
import com.vkt4u9999.myapp.databinding.ActivityMainBinding
import com.vkt4u9999.myapp.databinding.ActivityMainBinding.inflate

class MainActivity : AppCompatActivity() {

    private lateinit var mBinding: ActivityMainBinding


    private lateinit var test1: String

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        mBinding = inflate(layoutInflater)
        setContentView(mBinding.root)


        val items = listOf("apple, banana, orange")
       //mutableListOf("apple, banana, orange")

        for (item in items){
            println(item)
        }


        //while
        var index= 0
        while (index< items.size){
            println("Item at $index is ${items[index]}")
            index++
        }


    }
}
